### ISSUE TITLE
SOS python select on values with quotes

### ISSUE THREAD

**Issue by bschwal** [ovis-hpc/sos#43]
I was storing data as a char_array that had quotation marks around some values (i.e. "my_value").
When I used sos_cmd to select this data using sos_cmd -C my_path -qS schema -X index -F value:eq:'"my_value"'  (note the extra quotations around my_value) that worked just fine. 
When I tried to use a select statement in python with a where command ([value, Sos.Cond_EQ, '"my_value"']) this did not return any data. 
When I removed the quotation marks from the data before SOS ingest, the unaltered python code did return data. 

TLDR: SosDataSource.select is not filtering data correctly when there are quotation marks around some value.
